Target framework "netstandard2.0" does not build with error `The name 'Registry' does not exist in the current context`

### Prerequisites

- [x] Write a descriptive title.
- [x] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/Microsoft.PowerShell.PSResourceGet)
- [x] Search the existing issues.

### Steps to reproduce

* Clone repo
* `& '.\build.ps1' -BuildConfiguration 'Debug' -BuildFramework netstandard2.0 -Build -Clean`

```text
error CS0103: The name 'Registry' does not exist in the current context [D:\Git\personal\PowerShell--PSResourceGet\src\code\Microsoft.PowerShell.PSResourceGet.csproj::TargetFramework=netstandard2.0]
```

### Expected behavior

```console
Should build
```

### Actual behavior

```console
Does not build
```

### Error details

```console

```

### Environment data

```powershell
Windows 11 24H2 x64
PowerShell v7.4.6
```

### Visuals

<img width="913" alt="Image" src="https://github.com/user-attachments/assets/f95aff94-2c7d-430b-93fc-97eb3be339f9" />

--- a/build.ps1
+++ b/build.ps1
@@ -23,7 +23,7 @@ param (
     [ValidateSet("Debug", "Release")]
     [string] $BuildConfiguration = "Debug",
 
-    [ValidateSet("netstandard2.0", "net472")]
+    [ValidateSet("net472")]
     [string] $BuildFramework = "net472"
 )
 

--- a/src/code/Microsoft.PowerShell.PSResourceGet.csproj
+++ b/src/code/Microsoft.PowerShell.PSResourceGet.csproj
@@ -8,7 +8,7 @@
     <AssemblyVersion>1.1.0.1</AssemblyVersion>
     <FileVersion>1.1.0.1</FileVersion>
     <InformationalVersion>1.1.0.1</InformationalVersion>
-    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
   </PropertyGroup>

